salt/transport/tcp.py

```diff
@@ -1206,7 +1206,7 @@ def _add_client_present(self, client):
             clients = self.present[id_]
             clients.add(client)
         else:
-            self.present[id_] = set([client])
+            self.present[id_] = {client}
```

Comment:
As for this, I'm not certain it should be a tuple. The set is never modified, but I don't grok the TCP transport well enough to know whether or not `self.present[id_]` is supposed to be mutable.